.github/workflows/PR-build.yml

```diff
@@ -0,0 +1,144 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+name: PR Build
+on:
+  pull_request:
+    branches: 
+      - main 
+    types: 
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths-ignore:
+      - '.github/**'
+      - '!.github/workflows/CI.yml'
+      - '!.github/workflows/CI-Operator.yml'
```

Comment:
We don't have either of these files

Reply:
Nice catch. Remove all of theses and add the related files again ....